gui.js

```diff
@@ -7502,15 +7503,40 @@ WardrobeMorph.prototype.updateList = function () {
 
     this.addContents(paintbutton);
 
+    importbutton = new PushButtonMorph(
+        this,
+        "importNew",
+        new SymbolMorph("import", 15)
+    );
+    importbutton.padding = 0;
+    importbutton.corner = 12;
+    importbutton.color = IDE_Morph.prototype.groupColor;
+    importbutton.highlightColor = IDE_Morph.prototype.frameColor.darker(50);
+    importbutton.pressColor = importbutton.highlightColor;
+    importbutton.labelMinExtent = new Point(36, 18);
+    importbutton.labelShadowOffset = new Point(-1, -1);
+    importbutton.labelShadowColor = importbutton.highlightColor;
+    importbutton.labelColor = TurtleIconMorph.prototype.labelColor;
+    importbutton.contrast = this.buttonContrast;
+    importbutton.drawNew();
+    importbutton.hint = "Select a costume from the media library";
```

Comment:
"Select a costume from the media library" is the standard message, I didn't wrote it : it was already in code (hint used for costumes and sounds menu items).